src/aktualizr_lite/main.cc

```diff
@@ -236,7 +236,8 @@ static int daemon_main(LiteClient &client, const bpo::variables_map &variables_m
   while (true) {
     LOG_INFO << "Refreshing target metadata";
     if (!client.primary->updateImagesMeta()) {
-      LOG_WARNING << "Unable to update latest metadata, using local copy";
+      LOG_WARNING << "Unable to update latest metadata";
+      continue;  // There's no point trying to look for an update
```

Comment:
Won't this cause this function to be called right after again? On the normal path the code was always calling 'std::this_thread::sleep_for(std::chrono::seconds(interval));' before iterating again, which is not the case here.

Reply:
yep - this isn't correct. i'll fix.

Reply:
force-push-update